js/app/interfaces/card.js

```diff
@@ -236,6 +239,24 @@ const Card = new Lang.Interface({
     },
 
     /**
+     * Method: add_style_classes_from_model
+     * Add style classes to the card depending on model properties
+     *
+     * This adds some CSS classes to cards based on model properties such as
+     * content type, that card implementations may want to use to customize the
+     * way they style themselves.
+     *
+     * Every card implementation's constructor should call this method after
+     * setting the construct properties.
+     */
+    add_style_classes_from_model: function () {
```

Comment:
I think you got the order flipped.

This comes after the  chain to parent.
https://github.com/endlessm/eos-knowledge-lib/blob/master/js/app/knowledge.js#L90

and this in card.js
```
    _interface_init: function () {
        printerr(this.model);
    },
```
seems to print out models